### PR TITLE
TEXT-172: Failing test case to escape apastrophe

### PR DESCRIPTION
**Pull request by govi20**
Failing test case for the JIRA TEXT-172

### CODE REVIEW

File: src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java [hunk 1]
```diff
@@ -319,6 +319,12 @@ public void testUnescapeHtml4() {
         assertEquals("Hello&##;World", StringEscapeUtils.unescapeHtml4("Hello&##;World"));
     }
 
+    @Test
+    public void testEscapeHtml4Apostrophe() {
+        String unescapedChar = StringEscapeUtils.unescapeHtml4 ("&#39;");
+        assertEquals ("&#39;",unescapedChar, StringEscapeUtils.escapeHtml4 (unescapedChar));
```
Comment (Stzx) [inline src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java]: This test must fail.

Comment (govi20) [reply src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java]: Yeah, it failed in the Travis build.
org.opentest4j.AssertionFailedError: ' ==> expected: <&#39;> but was: <'>

File: src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java [hunk 2]
```diff
@@ -319,6 +319,12 @@ public void testUnescapeHtml4() {
         assertEquals("Hello&##;World", StringEscapeUtils.unescapeHtml4("Hello&##;World"));
     }
 
+    @Test
+    public void testEscapeHtml4Apostrophe() {
+        String unescapedChar = StringEscapeUtils.unescapeHtml4 ("&#39;");
```
Comment (Stzx) [inline src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java]: Try to use `final`.

Comment (govi20) [reply src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java]: Why?

File: src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java (hunk 1 above)
Comment (govi20) [reply src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java]: The test case is correct. `assertEquals ("&#39;", StringEscapeUtils.escapeHtml4 (unescapedChar));`

**Comment by govi20** [comment]
Closing the PR. As the corresponding JIRA has been closed.

**Pull request closed without merging**